package.json

```diff
@@ -36,7 +38,8 @@
     "react-native-safe-area-context": "^4.5.0",
     "react-native-screens": "^3.20.0",
     "react-native-svg": "^13.8.0",
-    "react-native-vector-icons": "^9.2.0"
+    "react-native-vector-icons": "^9.2.0",
+    "yarn": "^1.22.19"
```

Comment:
Do we need this?

Reply:
It got added when yarn is done. I removed it as it is not needed.